components/06-examples/samenwerken.njk

```diff
@@ -1,15 +1,15 @@
-{{> '@header' }}
+{% render '@header', {menu: menu, breadcrumbs: []} -%}
 <main>
   <div class="row">
     <div class="col-md-12">
       <h1>Samenwerken aan behandelen</h1>
-      {{> '@steps'}}
+      {% render '@steps', {title: title} -%}
     </div>
   </div>
   <div class="row">
     <div class="col-md-3">
-      {{> '@button' label='samenwerking toevoegen' type='button' modifier='link' icon='plus'}}
-    </div> 
+      {% render '@button', {label:'samenwerking toevoegen', type:'button', modifier:'link', icon:'plus'} %}
```

Comment:
Waar is de witruimte tussen de property en value gebleven? `{prop: 'value'} -> `{prop:'value'}`. We begonnen namelijk met whitespace